packages/maskbook/src/plugins/FileService/database.ts

```diff
@@ -8,13 +8,23 @@ type TaggedTypes = FileInfo
 const Database = createPluginDatabase<TaggedTypes>(identifier)
 
 export async function getRecentFiles() {
-    const files: FileInfo[] = await asyncIteratorToArray(Database.iterate('arweave'))
-    files.sort((a, b) => b.createdAt.getTime() - a.createdAt.getTime())
-    return files.slice(0, 4)
+    const skynet: FileInfo[] = await asyncIteratorToArray(Database.iterate('skynet'))
+    const arweave: FileInfo[] = await asyncIteratorToArray(Database.iterate('arweave'))
+    if (skynet != null) {
```

Comment:
why not concat `arweave` and `skynet` two collection?

Reply:
e.g:
```ts
const files = _.concat(arweave ?? [], skynet ?? [])
files.sort((a, b) => b.createdAt.getTime() - a.createdAt.getTime())
return files.slice(0, 4)
```

Reply:
Thanks, I will do.